blobs.go

```diff
@@ -185,6 +190,16 @@ type BlobWriter interface {
 	Reader() (io.ReadCloser, error)
 }
 
+// BlobKeeper combines the operations needed for registry's upkeeping such as
+// blob access, read, enumeration and delete. Can be used to prune unnecessary
+// objects.
+type BlobKeeper interface {
```

Comment:
I'm not thrilled about adding another inclusive interface into the `blobs` suite.  I peronally feel like `BlobStore` should be the all-encompassing type for all blob operations.